src/Microsoft.Health.SqlServer/Configs/SqlServerConnectionType.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.SqlServer.Configs
+{
+    public enum SqlServerConnectionType
+    {
+        ManagedIdentity,
+        WindowsIntegratedAuth,
+        Default,
```

Comment:
What is the default connection type? Can we give it a name like SqlUserNameAndPassword?

Reply:
Yeah, I was rather confused with the name so left it Default. I will change it to SqlUserNameAndPassword as it makes more sense.

Reply:
Updated.